public/templates/about.html

```diff
@@ -1,20 +1,29 @@
 {{>header this}}
 <main class="about-container">
-  <h1>Websurfx</h1>
-  <small
-    >a lightening fast, privacy respecting, secure meta search engine</small
-  >
-  <article>
-    Lorem ipsum dolor sit amet, officia excepteur ex fugiat reprehenderit enim
-    labore culpa sint ad nisi Lorem pariatur mollit ex esse exercitation amet.
-    Nisi anim cupidatat excepteur officia. Reprehenderit nostrud nostrud ipsum
-    Lorem est aliquip amet voluptate voluptate dolor minim nulla est proident.
-    Nostrud officia pariatur ut officia. Sit irure elit esse ea nulla sunt ex
-    occaecat reprehenderit commodo officia dolor Lorem duis laboris cupidatat
-    officia voluptate. Culpa proident adipisicing id nulla nisi laboris ex in
-    Lorem sunt duis officia eiusmod. Aliqua reprehenderit commodo ex non
-    excepteur duis sunt velit enim. Voluptate laboris sint cupidatat ullamco ut
-    ea consectetur et est culpa et culpa duis.
+  <article >
+    <div>
+    <h1 >Websurfx</h1>
+    <hr size="4" width="100%" color="#a6e3a1"> 
+    </div>
+    <p>A modern-looking, lightning-fast, privacy-respecting, secure meta search engine written in Rust. It provides a fast and secure search experience while respecting user privacy.<br> It aggregates results from multiple search engines and presents them in an unbiased manner, filtering out trackers and ads.
+    </p>
+
+    <h2>Some of the Top Features:</h2>
+
+    <ul><strong>Lightning fast </strong>- Results load within milliseconds for an instant search experience.</ul>
```

Comment:
One concern I have is that it seems some of the features listed here don't seem to be on the way right now, but longer term goals. I'm not sure it is appropriate to include them on this list at this time. @neon-mmd what is your opinion?

Reply:
I just took the points for the features part of readme.md and expanded them.

Reply:
I think having the feature list will be good, otherwise, the About page will look a little empty which will not look very nice, also I think this change will be good for now for the v1.0.0 release though I can see having changes in this in the future. for now, I think this is good enough. What do you say @XFFXFF ??

Reply:
Agreed, let's merge it!